$NetBSD$

* use gzip compression instead of zstd since libarchive might miss zstd even
  if QT uses it, fixes the configure step on NetBSD

--- src/corelib/CMakeLists.txt.orig	2021-01-26 18:29:22.000000000 +0000
+++ src/corelib/CMakeLists.txt
@@ -1179,7 +1179,7 @@ if(QT_FEATURE_mimetype AND QT_FEATURE_mi
             COMMENT "Generating ${qmimeprovider_db_output}"
         )
     else()
-        if(QT_FEATURE_zstd)
+        if(FALSE)
             set(qmime_db_compression Zstd)
             string(APPEND qmime_db_content "#define MIME_DATABASE_IS_ZSTD\n")
         else()
